.gitignore

```diff
@@ -0,0 +1,4 @@
+venv
+pycache
+idea
+sqlite3
```

Comment:
Add empty line

Reply:
@ArsenPidhoretskyi  why pycache shouldn't be?

Reply:
they are useless files